core/src/main/java/hudson/diagnosis/HudsonHomeDiskUsageMonitor.java

```diff
@@ -100,7 +100,7 @@ public static HudsonHomeDiskUsageMonitor get() {
     /**
      * Extension point for suggesting solutions for full JENKINS_HOME.
      *
-     * <h3>Views</h3>
+     * <h2>Views</h2>
```

Comment:
Perhaps these Javadoc fixes would be useful as a separate PR into `master`?

Reply:
Some of them applied cleanly to `master`, but others didn't. I'll try and sift through them at some point and proactively push what can be pushed now.